css/products.css

```diff
@@ -0,0 +1,200 @@
+/* single-column layout */
+body {
+    display: grid;
+    grid-template-areas: "header"
+        "nav"
+        "main"
+        "footer";
+    grid-template-rows: auto var(--size-15) 1fr auto;
+    grid-template-columns: 1fr;
+    min-height: 100vh;
+}
+
+/* header */
+.header {
+    grid-area: header;
+}
+
+/* aside-navbar for filters*/
+.aside {
+    grid-area: nav;
+    background-color: var(--dark-grey-color);
+    position: relative;
+}
+
+.aside-navbar {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--white);
+    transform: translateX(-100%);
+    transition: transform 560ms;
+    padding-bottom: var(--size-13);
+}
+
+.aside-navbar.show {
+    transform: translateX(0);
+}
+
+.beautify-aside {
+    z-index: 5;
+    background-color: rgba(255, 253, 253, 0.986);
```

Comment:
You can use a variable here and throughout the code

Reply:
I have implemented the changes.